### PR TITLE
Small fixes in the Jedis and BinaryJedis classes

### PR DESCRIPTION
**Pull request by pechenoha**

### CODE REVIEW

File: src/main/java/redis/clients/jedis/BinaryJedis.java [hunk 1]
```diff
@@ -252,11 +252,11 @@ public String quit() {
...
 
   /**
-   * Test if the specified keys exist. The command returns the number of keys existed Time
-   * complexity: O(N)
+   * Test if the specified keys exist. The command returns the number of keys exist.
+   * Time complexity: O(N)
    * @param keys
-   * @return Integer reply, specifically: an integer greater than 0 if one or more keys existed 0 if
-   *         none of the specified keys existed
+   * @return Integer reply, specifically: an integer greater than 0 if one or more keys exist
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/BinaryJedis.java]: Add a full stop at the end.

File: src/main/java/redis/clients/jedis/Jedis.java [hunk 2]
```diff
@@ -180,11 +180,11 @@ public String get(final String key) {
...
 
   /**
-   * Test if the specified key exists. The command returns the number of keys existed Time
-   * complexity: O(N)
+   * Test if the specified keys exist. The command returns the number of keys exist.
+   * Time complexity: O(N)
    * @param keys
-   * @return Integer reply, specifically: an integer greater than 0 if one or more keys were removed
-   *         0 if none of the specified key existed
+   * @return Integer reply, specifically: an integer greater than 0 if one or more keys exist
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/Jedis.java]: Add a full stop at the end.

File: src/main/java/redis/clients/jedis/BinaryJedis.java (hunk 1 above)
Comment (pechenoha) [reply src/main/java/redis/clients/jedis/BinaryJedis.java]: I suppose the full stop should be at the end of the next line (258), but the comma - at the end of this line (258), right?

File: src/main/java/redis/clients/jedis/Jedis.java (hunk 2 above)
Comment (pechenoha) [reply src/main/java/redis/clients/jedis/Jedis.java]: The same question as above.

File: src/main/java/redis/clients/jedis/BinaryJedis.java (hunk 1 above)
Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedis.java]: What we need is a proper punctuation after 258's `exist`. So I suggested a full stop. But your suggestion also works.

**Comment by sazzad16** [comment]
LGTM!

**Comment by sazzad16** [comment]
Merged into master, 2.10, 2.9.

**Pull request merged**